### PR TITLE
Item component / design FE

### PR DESCRIPTION
**Pull request by Hadeel-Salamin**
- [x] add /item path for Item component in App.js

- [x] create basic Item component with static data

- [x] do the style to make it look like the one in Figma

### CODE REVIEW

File: client/src/components/Item/item.style.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  color: #4c5268;
+`;
+
+export const ItemMainData = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+`;
+
+export const ItemSubtitle = styled.p`
+  font-family: Assistant;
```
Comment (jema28) [inline client/src/components/Item/item.style.js]: This is already in the global css body. This comment is for all the font family declarations.

File: client/src/components/Item/item.style.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+export const Line = styled.div`
+  height: 100px;
+  width: 5px;
+  background-color: #1ed390;
+  margin-right: 10px;
+`;
+
+export const ItemTitle = styled.p`
+  font-family: Assistant;
+  font-style: normal;
```
Comment (jema28) [inline client/src/components/Item/item.style.js]: You don't need normal here as it defaults as normal.

File: client/src/components/Item/item.style.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  font-weight: 600;
+  font-size: 14px;
+  line-height: normal;
+  color: #000000;
+`;
+export const ItemText = styled.p`
+  margin-left: 3px;
+  font-family: Assistant;
+  font-style: normal;
```
Comment (jema28) [inline client/src/components/Item/item.style.js]: Same here, see comment above.

File: client/src/components/Item/index.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+} from './item.style';
+
+class GetDetails extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <React.Fragment>
```
Comment (jema28) [inline client/src/components/Item/index.js]: You don't need this extra React Fragment as you're using a div.

**Review by jema28 — CHANGES_REQUESTED** [review]
Have a look at the comments and then ready to merge, great job 💯

**Review by FidaaHersh — APPROVED** [review]

File: client/src/components/Item/index.js (hunk 4 above)
Comment (Hadeel-Salamin) [reply client/src/components/Item/index.js]: removed

File: client/src/components/Item/item.style.js (hunk 3 above)
Comment (Hadeel-Salamin) [reply client/src/components/Item/item.style.js]: removed

File: client/src/components/Item/item.style.js (hunk 2 above)
Comment (Hadeel-Salamin) [reply client/src/components/Item/item.style.js]: removed

File: client/src/components/Item/item.style.js (hunk 1 above)
Comment (Hadeel-Salamin) [reply client/src/components/Item/item.style.js]: removed

**Review by jema28 — APPROVED** [review]

**Review by amanshawar — APPROVED** [review]

**Review by orjwan98 — APPROVED** [review]

**Pull request merged**